src/tiled/changeproperties.h

```diff
@@ -88,6 +89,7 @@ class SetProperty : public QUndoCommand
     QList<Object*> mObjects;
     QString mName;
     QString mValue;
+    QVariant::Type mType;       // The type of the property
```

Comment:
Here again, I believe it will be easier to just change the type of `mValue` to `QVariant`.